artifacts/rhdh-plugins-reference/tekton/tekton-plugin-admin.adoc

```diff
@@ -9,7 +9,7 @@ You can use the Tekton plugin to visualize the results of CI/CD pipeline runs on
 .Prerequsites
 * You have installed and configured the `@backstage/plugin-kubernetes` and `@backstage/plugin-kubernetes-backend` dynamic plugins.
 
-For more information about installing dynamic plugins, see xref:{installing-and-viewing-dynamic-plugins-url}[installing-and-viewing-dynamic-plugins-title].
+For more information about installing dynamic plugins, see link:{installing-and-viewing-dynamic-plugins-url}[{installing-and-viewing-dynamic-plugins-title}].
```

Comment:
@pabel-rh I'd suggest just having this as a follow-on sentence to `You have installed and configured the `@backstage/plugin-kubernetes` and `@backstage/plugin-kubernetes-backend` dynamic plugins.` - as it stands, it breaks up the unordered list of prerequisites and it's related to that first prerequisite.

Reply:
It is out of scope for a backport pull request.